src/core/dev_api/openvino/op/group_query_attention.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+
+#include "openvino/op/op.hpp"
+
+namespace ov {
+namespace op {
```

Comment:
```suggestion
namespace ov::op::internal {
```
Put it in internal namespace or give version